CMakeLists.txt

```diff
@@ -58,26 +58,15 @@ if (USE_SYSTEM_ARCH MATCHES LINUX)
 
     # Configure files for find_package
     configure_file(
-        ${PROJECT_SOURCE_DIR}/pkg/payloadsdk-config.cmake.in
```

Comment:
what is the reason that we need change the folder name from "pkg" to "cmake"?  If we need make our fork branch to be easily merge with DJI's next version, its better to keep their naming, except there is bug or error.  What do you think?